### DISCUSSION TITLE
YouTube Video Background for Section

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#2172] [Q&A]
I'm running Vanjaro 1.3 as an extension on DNN 9.10.2. I added a section and wanted to add a video background from YouTube. I've added the API Key for Youtube, as I'm able to search online for video using the Video Block. 

Can a video from YouTube be added as a background video or does the video have to reside locally? When I try to add a video background, I'm only seeing a File System and don't have a tab to search online for YouTube videos. 

Am I missing something?

**Comment by meetmandeep** [comment]
You cannot use YouTube videos for Section Backgrounds. Search Online tab should appear if you have Pixabay integration configured.
